### PR TITLE
Feat simple amos nft contract

### LINKED ISSUE #39: Solidity Contract for NFTs
## User story
1. As a poweruser
2. I want / need a underlying contract 
3. So that it can carry my NFT

## Acceptance criteria
* Create Solidity Contract which is able to carry a NFT

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

### PR DESCRIPTION
**Pull request by phildree**
- We implemented a new contract which is now being used.
- The new contract now supports to add a link (to a picture) and a hash
- The new contract enables the user to retrieve the stored values (URL & hash)

closes #39 
It would be preferable to merge this PR before implementing changes in the StartMintingCommand File, since we had to add some modifications @laila-rin @tawfeeka @lukas-rotsching

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,4 +29,4 @@
 
 ## Our really nice logo! 🚀
 
-![ape](https://user-images.githubusercontent.com/92869397/166645877-e8570f35-82fd-41cb-a702-3b5d1a3068a0.JPG)
+![ape](https://user-images.githubusercontent.com/92869397/166645877-e8570f35-82fd-41cb-a702-3b5d1a3068a0.JPG)
```
Comment (lukas-rotsching) [inline README.md:32]: what is this change? I don't see a difference.

**Review by tawfeeka — APPROVED** [review]
look great

**Review by lukas-rotsching — COMMENTED** [review]
we moved your code in StartMintingCommand.ts:execute() into its own method. Please do not move it back to execute() ! You did that in this commit 38a184e69f58b1409c7006d5cf3eac49dbfa9964 . That commit was part of your previous and already merged pull request. But for the future: Please don't do this!
 If your code stays in its own method we won't have any merge conflicts. It's not of your concern how the execution of your code in the frontend is structured.

**Comment by schlingling** [comment]
> we moved your code in StartMintingCommand.ts:execute() into its own method. Please do not move it back to execute() ! You did that in this commit [38a184e](https://github.com/amosproj/amos2022ss07-nft-playbook/commit/38a184e69f58b1409c7006d5cf3eac49dbfa9964) . That commit was part of your previous and already merged pull request. But for the future: Please don't do this!
> If your code stays in its own method we won't have any merge conflicts. It's not of your concern how the execution of your code in the frontend is structured.

Okay @lukas-rotsching we will keep this in mind by our next PR

**Comment by phildree** [comment]
This was necessary for our testing.  We really need to set our repo up in another/the correct way, so that we are able to execute our code without changing anything from your code. This is definitely something we need to discuss in the next meeting. @lukas-rotsching @tawfeeka

**Pull request merged**